### PR TITLE
[WIP] Crew Objectives: super shitcode edition

### PR DESCRIPTION
**Pull request by Jujumatic**
So I've seemingly made the skeleton for crew objectives thus far, had to spend a few days reading datums and looking around the codebase to see what the hell I even needed to do but I think I've got a good idea of what needs to be done. This is probably going to take a while but lol

Give some crew objectives, whether or not it's restricted to a job and what it should be

As it currently stands, crew objectives will only be given on roundstart itself. That can probably be fixed pretty easily though.

Already an approved feature so votes are not needed I don't think

:cl:
add: Added crew objectives!!! Objectives are created for all non-antags and can be generic or based on your job type.
/:cl:

### CODE REVIEW

File: hippiestation/code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -54,4 +54,40 @@
...
+	explanation_text = "Something something"
+	var/job_required = ""
+
+/datum/objective/crew/clown/honk
+	job_required = "Clown"
+
+/datum/objective/crew/clown/honk/update_explanation_text()
+	explanation_text = "Honk 50 people with your bike horn!"
+
+/datum/objective/crew/clown/honk/check_completion()
```
Comment (Jujumatic) [inline hippiestation/code/game/gamemodes/objective.dm:69]: idk how to check for this yet lol

**Comment by Carbonhell** [comment]
If there is a feature request for this in github issue tracker, add "Fixes #issuenum" to the PR description

File: hippiestation/code/modules/antagonists/crew/crew.dm [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/datum/antagonist/crew
+	name = "Crewmember"
+	var/datum/mind/M
+	show_in_antagpanel = FALSE
+	show_in_roundend = FALSE	//WE ARE GOING TO GET SPAMMED TO SHIT IF THIS IS TRUE
+
+/datum/antagonist/crew/New()
+	. = ..()
+	M = var/datum/antagonist/crew/C
```
Comment (Carbonhell) [inline hippiestation/code/modules/antagonists/crew/crew.dm:9]: wtf is this syntax

**Review by Carbonhell — CHANGES_REQUESTED** [review]

**Comment by Carbonhell** [comment]
Your syntax is wrong in multiple places

**Comment by Jujumatic** [comment]
Will re-open once I get this done properly

**Pull request closed without merging**